ksqldb-functional-tests/src/test/resources/query-validation-tests/formats.json

```diff
@@ -1,22 +1,276 @@
 {
   "tests": [
     {
-      "name": "invalid format name",
+      "name": "invalid value format",
       "statements": [
         "CREATE STREAM TEST WITH (kafka_topic='test_topic', value_format='FOO');"
       ],
-      "topics": [
-        {
-          "name": "test_topic",
-          "format": "JSON"
+      "expectedException": {
+        "type": "io.confluent.ksql.util.KsqlStatementException",
+        "message": "Unknown format: FOO"
+      }
+    },
+    {
+      "name": "missing value format",
+      "statements": [
+        "CREATE STREAM TEST WITH (kafka_topic='test_topic');"
+      ],
+      "expectedException": {
+        "type": "io.confluent.ksql.parser.exception.ParseFailedException",
+        "message": "Failed to prepare statement: Missing required property \"VALUE_FORMAT\" which has no default value."
+      }
+    },
+    {
+      "name": "key format without feature flag",
+      "statements": [
+        "CREATE STREAM TEST (foo VARCHAR) WITH (kafka_topic='test_topic', value_format='JSON', key_format='KAFKA');"
+      ],
+      "expectedException": {
+        "type": "io.confluent.ksql.util.KsqlStatementException",
+        "message": "The use of 'KEY_FORMAT' and 'FORMAT' is disabled, as this feature is under development."
+      }
+    },
```

Comment:
Todo: allow... right?

Reply:
Yup, now that we allow the use of `KEY_FORMAT` and `FORMAT` even when the feature flag is disabled I've removed these tests and also removed the explicit enabling of the feature flag from a couple other (otherwise redundant) tests in this file, and added historic plans for them accordingly.